pkg/console/subresource/configmap/configmap.go

```diff
@@ -18,23 +19,57 @@ const (
 	consoleConfigYamlFile   = "console-config.yaml"
 	clientSecretFilePath    = "/var/oauth-config/clientSecret"
 	oauthEndpointCAFilePath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-	// TODO: should this be configurable?  likely so.
-	documentationBaseURL = "https://docs.okd.io/4.0/"
-	brandingDefault      = "okd"
 	// serving info
 	certFilePath = "/var/serving-cert/tls.crt"
 	keyFilePath  = "/var/serving-cert/tls.key"
 )
 
-func DefaultConfigMap(cr *operatorv1.Console, rt *routev1.Route) *corev1.ConfigMap {
+// overridden by console config
+const (
+	defaultLogoutURL = ""
+)
+
+// overridden by operator config
+const (
+	defaultDocumentationBaseURL = "https://docs.okd.io/4.0/"
+	defaultBranding             = "okd"
+)
+
+//consoleConfig.Spec.Authentication.LogoutRedirect
+func getLogoutRedirect(consoleConfig *configv1.Console) string {
+	// TODO: handle if not provided
+	// return consoleConfig.Spec.Authentication.LogoutRedirect || defaultLogoutURL
```

Comment:
Lets uncomment this and comment the below ? or am I missing something ?

Reply:
Nope, you aren't missing anything. This is why its still WIP.  I didn't have time to test everything before opening last night, but wanted to raise awareness early.